components/StyledButton/index.js

```diff
@@ -45,4 +44,20 @@ export const TransparentFavoriteButton = styled.button`
   gap: 5px;
   z-index: 1;
   cursor: pointer;
-`;
\ No newline at end of file
+`;
+
+export const TransparentBackButton = styled.button`
+  position: absolute;
+  top: 10px;
+  left: 10px;
+  background: rgb(255 255 255 / 80%);
+  color: black;
+  padding: 0.3125rem;
+  border-radius: 50%;
```

Comment:
as long as your button is quadratic this is fine, but as soon as one side is getting longer than the other, your button wiil have an awkward oval shape. To accommodate that I would recommend to use a very high pixel value, like 999px for example. This will result in a "pill" shape, if one side is longer than the other.